Outline script fails

### ✅ Have you read and understood the above guidelines?

yes

### 📜 What is the name of the script you are using?

Outline

### 📂 What was the exact command used to execute the script?

bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/outline.sh)"

### 📝 Provide a clear and concise description of the issue.

Hi, installation fails with this error:

[ERROR] in line 74: exit code 0: while executing command $STD yarn build

### ⚙️ What settings are you using?

- [ ] Default Settings
- [x] Advanced Settings

### 🖥️ Which Linux distribution are you using?

Debian 12

### 🔄 Steps to reproduce the issue.

Install script, verbose mode, using advanced settings ONLY to change CT id, root pwd, CT/Gateway IPs.
NO TOUCHING resources like RAM, cores, or any other setting except the ones above.
Debian 12 unprivileged.

### ❌ Paste the full error output (if available).

[5/5] Building fresh packages...
$ yarn patch-package
yarn run v1.22.22
$ /opt/outline/node_modules/.bin/patch-package
 ⠇patch-package 7.0.2
Applying patches...
@benrbray/prosemirror-math@0.2.2 ✔
Done in 0.12s.
$ husky install
fatal: not a git repository (or any of the parent directories): .git
husky - git command not found, skipping install
Done in 98.26s.
yarn run v1.22.22
$ yarn clean && yarn vite:build && yarn build:i18n && yarn build:server
$ rimraf build
$ VITE_CJS_IGNORE_WARNING=true vite build
 ⠏vite v5.4.12 building for production...
node_modules/sonner/dist/index.mjs (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/sonner/dist/index.mjs" was ignored.
node_modules/react-medium-image-zoom/dist/index.js (1:0) Module level directives cause errors when bundled, "use client" in "node_modules/react-medium-image-zoom/dist/index.js" was ignored.
✓ 7216 modules transformed.
 ⠋(webpackStats plugin) Stats saved to /opt/outline/build/app/webpack-stats.json (567.875KiB)
 ⠏Killed
error Command failed with exit code 137.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 137.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
 ⠙
[ERROR] in line 74: exit code 0: while executing command $STD yarn build

### 🖼️ Additional context (optional).

_No response_

similar issue here:

🧩  Using Advanced Settings on node pve03
  🖥️  Operating System: debian
  🌟  Version: 12
  📦  Container Type: Unprivileged
  🔐  Root Password: ********
  🆔  Container ID: 101
  🏠  Hostname: outline
  💾  Disk Size: 12 GB
  🧠  CPU Cores: 2
  🛠️  RAM Size: 4096 MiB
  🌉  Bridge: vmbr0
  📡  IP Address: dhcp
  🌐  Gateway IP Address: Default
  📡  APT-Cacher IP Address: Default
  🚫  Disable IPv6: no
  ⚙️  Interface MTU Size: Default
  🔍  DNS Search Domain: Host
  📡  DNS Server IP Address: Host
  🏷️  Vlan: Default
  📡  Tags: community-script;documentation
  🔑  Root SSH Access: yes
Warning: No SSH key provided.
  🔍  Verbose Mode: no
  🚀  Creating a Outline LXC using the above advanced settings
  ✔️  Using local for Template Storage.
  ✔️  Using local-zfs for Container Storage.
  ✔️  Updated LXC Template List
  ✔️  LXC Template is ready to use.
  ✔️  LXC Container 101 was successfully created.
  ✔️  Started LXC Container
  ✔️  Set up Container OS
  ✔️  Network Connected: 10.12.22.249 fd00:10:12:22:be24:11ff:fe60:c7f7 
  ✔️  IPv4 Internet Connected
  ✖️  IPv6 Internet Not Connected
  ✔️  DNS Resolved github.com to 140.82.121.4
  ✔️  Updated Container OS
  ✔️  Installed Dependencies
  ✔️  Set up Node.js Repository
  ✔️  Set up PostgreSQL Repository
  ✔️  Installed Node.js
  ✔️  Set up PostgreSQL
 ⠹ Setup Outline (Patience) 
[ERROR] in line 50: exit code 0: while executing command "$@" > /dev/null 2>&1
root@pve03:~#

I can log into the console and edit the .env, but accessing the web interface fails.

--- a/install/outline-install.sh
+++ b/install/outline-install.sh
@@ -46,7 +46,7 @@ $STD apt-get install -y postgresql-16
 DB_NAME="outline"
 DB_USER="outline"
 DB_PASS="$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | cut -c1-13)"
-SECRET_KEY="$(openssl rand -base64 32 | tr -dc 'a-zA-Z0-9' | cut -c1-32)"
+SECRET_KEY="$(openssl rand -hex 32)"
 $STD sudo -u postgres psql -c "CREATE ROLE $DB_USER WITH LOGIN PASSWORD '$DB_PASS';"
 $STD sudo -u postgres psql -c "CREATE DATABASE $DB_NAME WITH OWNER $DB_USER ENCODING 'UTF8' TEMPLATE template0;"
 $STD sudo -u postgres psql -c "ALTER ROLE $DB_USER SET client_encoding TO 'utf8';"

